app/main/helpers/direct_award_helpers.py

```diff
@@ -0,0 +1,2 @@
+def is_direct_award_project_accessible(project, user_id):
+    return user_id in [user['id'] for user in project['users']]
```

Comment:
`any(user["id"] == user_id for user in project['users'])`

Reply:
Thanks. 👍